infra/testing/helpers/generateOpaqueResponse.mjs

```diff
@@ -0,0 +1,18 @@
+/*
+  Copyright 2019 Google LLC
+
+  Use of this source code is governed by an MIT-style
+  license that can be found in the LICENSE file or at
+  https://opensource.org/licenses/MIT.
+*/
+
+
+// Cache a resonse value and clone it instead of re-fetching every time.
+let response;
+
+export const generateOpaqueResponse = async () => {
+  if (!response) {
+    response = await fetch('https://google.com', {mode: 'no-cors'});
```

Comment:
I *thought* we used to start up two test servers on different ports, and then use the second when we wanted to simulate a cross-origin request. I am having a hard time tracking down when that changed, though.

Doing that again would be another option, and one that would allow us to keep everything "in house".

Reply:
Yeah, I think we could do this, but I'll wait to do it in a separate PR.